agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -519,6 +520,15 @@ class backhaul_manager : public btl::transport_socket_thread {
                           const sLinkNeighbor &link_neighbor, const sLinkMetrics &link_metrics,
                           ieee1905_1::eLinkMetricsType link_metrics_type);
 
+    struct sExpectedApMetricsResponse {
+        uint16_t mid;
+        std::vector<sMacAddr> expected_bssids;
+        uint8_t response_buffer[message::MESSAGE_BUFFER_LENGTH];
+        ieee1905_1::CmduMessageTx response = {response_buffer, sizeof(response_buffer)};
+    };
+
+    sExpectedApMetricsResponse m_expected_ap_metrics_response;
```

Comment:
```suggestion
    sApMetricsResponse m_ap_metrics_response_under_construction;
```
The same goes for the variable.